pandas/tests/reshape/test_melt.py

```diff
@@ -101,15 +101,25 @@ def test_vars_work_with_multiindex(self):
         tm.assert_frame_equal(result, expected)
 
     def test_single_vars_work_with_multiindex(self):
-        expected = DataFrame(
+        expected0 = DataFrame(
```

Comment:
can you parametrize this test instead

Reply:
At some point, all other tests should be parameterized too. This one's done